components/cdc/src/initializer.rs

```diff
@@ -186,6 +166,11 @@ impl<E: KvEngine> Initializer<E> {
     ) -> Result<()> {
         if let Some(region_snapshot) = resp.snapshot {
             assert_eq!(self.region_id, region_snapshot.get_region().get_id());
+            CDC_SCAN_TASKS.with_label_values(&["ongoing"]).inc();
```

Comment:
How about moving these 4 lines to the `async_incremental_scan`? That would enhance the cohesion of the `async_incremental_scan` function.